src/test/parse-fail/obsolete-proc.rs

```diff
@@ -10,8 +10,8 @@
 
 // Test that we generate obsolete syntax errors around usages of `proc`.
 
-fn foo(p: proc()) { } //~ ERROR obsolete syntax: the `proc` type
+fn foo(p: proc()) { } //~ ERROR: the `proc` type isn't used for the moment
```

Comment:
Should this error message be updated? I think the parser modifications you made earlier may have changed the error message (you can check with `make check-stage2-pfail`